File: frontend/lib/models/service.dart
class Service {
  final int id;
  final String name;
  final String description;
  final int durationMinutes;
  final int priceCents;

  Service({
    required this.id,
    required this.name,
    required this.description,
    required this.durationMinutes,
    required this.priceCents,
  });

  factory Service.fromJson(Map<String, dynamic> json) {
    return Service(
      id: json['id'] as int,
      name: json['name'] as String,
      description: json['description'] ?? '',
      durationMinutes: json['duration_minutes'] as int,
      priceCents: json['price_cents'] as int,
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'name': name,
      'description': description,
      'duration_minutes': durationMinutes,
      'price_cents': priceCents,
    };
  }
}

File: backend/models/service.py
from sqlalchemy import Column, Integer, String, Text, DateTime, func
from database import Base

class Service(Base):
    __tablename__ = "services"

    id = Column(Integer, primary_key=True, index=True)
    name = Column(String(100), nullable=False)
    description = Column(Text, nullable=True)
    duration_minutes = Column(Integer, nullable=False) # Must be % 15 == 0 in DDL
    price_cents = Column(Integer, nullable=False)
    created_at = Column(DateTime(timezone=True), server_default=func.now())
